src/estimators/stratified_sketch.py

```diff
@@ -23,53 +23,128 @@
 ONE_PLUS = '1+'
 
 
+class ExactSetOperator(object):
```

Comment:
I agree.  The SketchOperator will be introduced in #44 . To avoid conflict and redefinition, I suggest keeping it this way and do a cleanup after #44 is submitted. What do you think?

Reply:
sgtm. Please add a TODO before submitting.